changelogs/unreleased/5562-autostarted-agents-log.yml

```diff
@@ -0,0 +1,6 @@
+description: Autostarted agents will now log in debug mode
+issue-nr: 5562
+change-type: patch
+destination-branches: [master, iso6, iso5]
```

Comment:
I think this should go into ISO4 as well. Having DEBUG logs on the agent is quite important at debugging time.